### DISCUSSION TITLE
Support for visionOS

### DISCUSSION THREAD

**Discussion by darkroastcreative** [MobiusSync/MobiusSync#144] [Ideas]
I’d like to suggest/request that Möbius Sync support visionOS to increase coverage across different devices and platforms in the Apple ecosystem (even if this is just a matter of offering Möbius Sync as a [“Compatible App”](https://developer.apple.com/documentation/visionos/making-your-app-compatible-with-visionos)).

From personal experience with visionOS, the absence of Möbius Sync on the Apple Vision platform has been an impediment to being able to take full advantage of the platform and its accompanying hardware (a testament to the value the software provides currently on iOS and iPadOS). I’ve tried a few “workarounds,” but they haven’t been nearly as seamless or reliable as Möbius Sync. I’d even happy pay for this “feature” (even if it was separate from Möbius Sync Pro, which I currently have).

I recognize that visionOS is very new and there may not be a large user base just yet, but I hope this suggestion may at least start a conversation that helps Möbius Sync grow and continue to live up to its objective to “Sync files between **all** your devices.”

**Comment by SimonPickup** [comment]
See https://github.com/MobiusSync/MobiusSync/issues/141
